src/unix/internal.h

```diff
@@ -492,6 +492,7 @@ uv__fs_copy_file_range(int fd_in,
 typedef struct {
   long long quota_per_period;
   long long period_length;
+  long long quota;
   double proportions;
 } uv__cpu_constraint;
```

Comment:
I'd simply remove this struct and return `quota` directly. Avoids confusion about what fields are and aren't used.

Reply:
Done